app/(home)/(posts)/post/[slug]/page.tsx

```diff
@@ -83,41 +85,55 @@ export default function PostPage({
   };
 
   const getPost = async (stream_id: string): Promise<void> => {
+    if (!window.ethereum) {
+      console.error("No Ethereum provider found");
+      return;
```

Comment:
Is there a difference between using `return;` (here) and `return undefined;` (L140)?

This same scenario happens in other areas of the demo.

Reply:
> Is there a difference between using `return;` (here) and `return undefined;` (L140)?
> 
> This same scenario happens in other areas of the demo.

Actually `return;` and `return undefined;` is the same. I prefer just `return` because I find it more aesthetic, but the alternative is ok as well.

https://github.com/nucypher/taco-orbisdb-demo/pull/6